flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/rules/logical/PushFilterIntoLegacyTableSourceScanRuleTest.scala

```diff
@@ -196,4 +196,9 @@ class PushFilterIntoLegacyTableSourceScanRuleTest extends TableTestBase {
         |  TIMESTAMPADD(YEAR, 2, b) >= a
         |""".stripMargin)
   }
+
+  @Test
+  def testCannotPushDownIn(): Unit = {
```

Comment:
nit: use a meaning name which can describe the bug or add some comments to describe the bug